recipe/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+set -ex
+
+mkdir build
+cd build
+
+cmake -G Ninja \
+    ${CMAKE_ARGS} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_CXX_STANDARD=17 \
+    -DCMAKE_INSTALL_LIBDIR=lib \
+    -DBUILD_SHARED_LIBS=ON \
+    -DBUILD_DEPS=OFF \
+    -DBUILD_FLATZINC=ON \
+    -DBUILD_GLOP=OFF \
```

Comment:
since you have BUILD_CXX=ON (default to ON) then BUILD_GLOP is forced to OFF...

Reply:
I understood that, but this is more for the purpose of clarity to ensure that things are self-documenting in the build-script (also noted in the commit so it's easy to find through the git blame)